docs/immortal-empires/kislev.md

```diff
@@ -0,0 +1,32 @@
+# Kislev
+
+## Kostaltyn: The Great Orthodoxy
+
+* **Route I - There Can Only Be One**
+  * PLACEHOLDER:
+    * Erengrad, Praag, Kislev
+  * Have at least 600 Supporters
+  * **Reward:**
+    * _PLACEHOLDER_: Ice Court training cost + duration -X
+
+* **Route II - The Last Bastion**
+  * PLACEHOLDER
+  * Destroy factions:
+    * Clan Moulder, Azazel, Throgg, Wulfrik
+  * **Reward:**
+    * _PLACEHOLDER_: Melee defence +X% when under siege, campaign movement range: +X% for enemy armies starting their turn in your regions
```

Comment:
I thing melee defence when under siege can get quite crazy considering the bonus below. I still hink a shield of faith (barrier hit points, see The Northern Provinces) would work better here.

Reply:
Added